src/ScriptingBridge/SBApplication.cs

```diff
@@ -7,30 +9,36 @@
 namespace ScriptingBridge {
 	public partial class SBApplication
 	{
-#if XAMCORE_4_0
-		public static SBApplication GetApplication (string ident)  => Runtime.GetNSObject<SBApplication> (_FromBundleIdentifier (ident));
+		public static SBApplication? GetApplication (string ident)  => Runtime.GetNSObject<SBApplication> (_FromBundleIdentifier (ident));
```

Comment:
Should we be replacing this `#if XAMCORE_4_0` with an `#if NET`?

Reply:
Not if this is an API compatible improvement. We can check the api diff to see if Rolf is right here.

Reply:
The API diff shows only added methods: https://gist.github.com/vs-mobiletools-engineering-service2/8e637628f885a913bfa0dfe00c9e4482